website/src/pages/api/auth/[...nextauth].ts

```diff
@@ -33,7 +34,7 @@ if (process.env.DISCORD_CLIENT_ID) {
   );
 }
 
-if (process.env.NODE_ENV === "development") {
+if (boolean(process.env.DEBUG_LOGIN) || process.env.NODE_ENV === "development") {
```

Comment:
LGTM

The name `DEBUG_LOGIN` does not really represent what the variables does, maybe `USE_DUMMY_USER`, or `USE_DEFAULT_CREDENTIALS` or something similar?

Unless I misunderstood something...

Reply:
there's not really a dummy user or default credentials, it's the login itself that's in a debug configuration: it just takes any username and gives you a session as that user.

Reply:
aha, I see, so I did misunderstand it 😅